setup.py

```diff
@@ -6,6 +6,12 @@
 
 VERSION = '1.1.5'
 
+INSTALL_REQUIRES = (
+    'Orange3>=3.9.0',
```

Comment:
No, sorry, install Orange on Travis only. I promote this package as a generally useful association rules mining package. No need to tag the whole Orange and company everywhere.